src/features/preview/MachinePreview.js

```diff
@@ -20,13 +26,44 @@ const mapDispatchToProps = (dispatch, ownProps) => {
     onSlider: (value) => {
       dispatch(updatePreview({sliderValue: value}))
     },
+    onLayerChange: (attrs) => {
+      dispatch(updateLayer(attrs))
+    },
+    onKeyDown: (event, currentLayer) => {
+      let attrs = { id: currentLayer.id }
+
+      if (['ArrowDown', 'ArrowUp', 'ArrowLeft', 'ArrowRight'].includes(event.key)) {
+        const delta = !!event.shiftKey ? 1 : 5
```

Comment:
What is the !!? Is that not not?

The shift key seems to be highlighting text. I don't really mind, but I noticed in when I tried using shift+arrows.

Reply:
!! converts the value to a Boolean.

Reply:
:+1: